### PR TITLE
Fasthttp router support

### PR DESCRIPTION
**Pull request by Pradhyuman-sharma**
feat(integrations): adding support for fasthttp router

### CODE REVIEW

**Comment by slayerjain** [comment]
@Sarthak160 @re-Tick please review this

**Review by re-Tick — APPROVED** [review]

**Review by Sarthak160 — APPROVED** [review]

File: integrations/kfasthttp/fasthttp.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+		Body:       resBody.String(),
+	}
+
+}
+
+func setContextValFast(c *fasthttp.RequestCtx, val interface{}) {
+	c.SetUserValue(string(keploy.KCTX), val)
+
+}
+func Fast(k *keploy.Keploy) func(fasthttp.RequestHandler) fasthttp.RequestHandler {
```
Comment (slayerjain) [inline integrations/kfasthttp/fasthttp.go:49]: @Pradhyuman-sharma you're already in a package called `kfasthttp`. Why did you call this method `fast`?

File: integrations/kfasthttp/fasthttp.go [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+			})
+
+			r = r.WithContext(ctx)
+
+			keploy.CaptureTestcase(k, r, reqBody, resp, params)
+			return
+		})
+	}
+}
+func paramsfast(c *fasthttp.RequestCtx) map[string]string {
```
Comment (slayerjain) [inline integrations/kfasthttp/fasthttp.go:105]: @Pradhyuman-sharma rethink this method name too

**Review by slayerjain — CHANGES_REQUESTED** [review]
The PR looks good. Please just address the minor issues I commented upon

File: integrations/kfasthttp/fasthttp.go (hunk 1 above)
Comment (Pradhyuman-sharma) [reply integrations/kfasthttp/fasthttp.go:49]: ohh it is the middleware function which needs to be called at the time of integration like in chi router it is called as ChiMiddleware and so on in other routers.

File: integrations/kfasthttp/fasthttp.go (hunk 2 above)
Comment (Pradhyuman-sharma) [reply integrations/kfasthttp/fasthttp.go:105]: yes I can call it like pathparams too?

File: integrations/kfasthttp/fasthttp.go (hunk 1 above)
Comment (slayerjain) [reply integrations/kfasthttp/fasthttp.go:49]: That needs to be corrected too. It's better to call it middleware or just mw is also fine

Comment (Pradhyuman-sharma) [reply integrations/kfasthttp/fasthttp.go]: I just made some change please take a look.

**Pull request merged**